ansible_collections/arista/avd/roles/eos_designs/templates/base/main.j2

```diff
@@ -56,3 +56,5 @@
 {% include 'base/eos-cli.j2' %}
 
 {% include 'base/struct-cfg.j2' %}
+
+{% include 'base/is_deployed_shutdown_peer_interfaces.j2' %}
```

Comment:
`struc-cfg.j2` should be the latest included template (as this is custom_structured_configuration that should be able to override anything)

However - does this setting needs to be included in the structured_configuration? what is more important is to have the shutdown interfaces right? (like it is done in the facts)

Reply:
I didn't think about the ordering here but yes, I do believe it is more important to have the interfaces shutdown so any structured configs that enable the port should be overridden by this variable.

Reply:
so I don't necessarily agree - the principle of custom_struct is to be able to override values generated by `eos_designs` and hence it should be possible to override this one as well (and yes `eos_validate_state` would fail on some tests but at some point you need to know what you are doing when using custom_struct)

but my second point was actually maybe more relevant and making the first one moot - looking at the template `is_deployed_shutdown_peer_interfaces.j2` it is actually putting in the `intended/structured_configuration` a variable that is not going to be used afterwards - is it actually needed here? Basically you have already used in the facts the value to decide to shutdown or not the interfaces right? So even if it is overriden at this stage - it is "already too late" because the facts have already generated the configuration with the interfaces toward the undeployed peers shutdown.

Reply:
Understood, it's a unnecessary variable at this stage :). I'll remove the j2 file and update main.j2.